lib/puppet/parser/functions/validate_file_exists.rb

```diff
@@ -10,7 +10,7 @@ module Puppet::Parser::Functions
   ) do |args|
 
     args.each do |arg|
-      raise Puppet::ParseError, "does not exist" unless File.exist?(arg)
+      raise Puppet::ParseError, "does not exist" unless File.exists?(arg)
```

Comment:
They're identical, except in name.

However, if I use a `File.expects(:exist?).yada.yada.yada.once` in the spec, some of the calls by rspec-puppet itself trigger it and it breaks the spec.  `exists?` does not.

I tried a bunch of ways to avoid this, fakefs (doesn't work with rspec-puppet), mocks, etc, but this was the cleanest to just add an `s`.  puppetlabs-stdlib specs do the same.